conf/cqlshrc.sample

```diff
@@ -33,6 +32,10 @@
 ; classname = PlainTextAuthProvider
 ; username = user1
 
+[protocol]
```

Comment:
@neshkeev is this really relevant? can you point me to the PR suggesting to add this?